File: src/main/java/org/example/gogoma/domain/usermarathon/dto/UserMarathonDetailDto.java
package org.example.gogoma.domain.usermarathon.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import org.example.gogoma.domain.marathon.entity.Marathon;
import org.example.gogoma.domain.usermarathon.enums.PaymentType;

import java.time.LocalDateTime;
import java.util.List;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UserMarathonDetailDto {

    private Marathon marathon;
    private List<String> courseTypeList;
    private PaymentType paymentType;
    private String paymentAmount;
    private LocalDateTime paymentDateTime;
    private String address;
    private String selectedCourseType;
    private int targetPace;
}

File: app/src/main/java/com/example/gogoma/data/dto/UserMarathonDetailDto.kt
package com.example.gogoma.data.dto

import android.os.Parcelable
import com.example.gogoma.data.model.PaymentType
import com.example.gogoma.data.model.Marathon
import kotlinx.parcelize.Parcelize

@Parcelize
data class UserMarathonDetailDto (
    val marathon: Marathon,

    val courseTypeList: List<String>,

    val paymentType: PaymentType,

    val paymentAmount: String,

    val paymentDateTime: String,

    val address: String,

    val selectedCourseType: String
) : Parcelable